### PR TITLE
Exercise 3 Draft 1 review

### LINKED ISSUE #1: Exercise 3 Draft 1 - Review
Issue to track exercise 3 reviews.
@don-morrison-2000 FYI

### PR DESCRIPTION
**Pull request by marcelaadan1**
Ex 3 draft 1 commit
Please review and comment.

### CODE REVIEW

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md [hunk 1]
```diff
@@ -65,6 +65,37 @@ Complete the following tasks:
...
+
+![](../.gitbook/assets/ex3/image005.jpg)
+
+* [ ] Accept the default values for service name and resource group.
+* [ ] In **Available authentication methods**, select **Use only IAM**, as shown in the following figure. 
+
+![](../.gitbook/assets/ex3/image006.jpg)
+
+{% hint style="info" %}
+:memo: The Cloudant service has two authentication methods that are available: **Use both legacy credentials and IAM**, which means that both IAM and Legacy credentials can be used to access the account, and Use only IAM, which means that only IAM credentials are provided through service binding and credential generation.
```
Comment (don-morrison-2000) [inline workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md:79]: This is a good memo. But I can't see the final form

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md [hunk 2]
```diff
@@ -65,6 +65,37 @@ Complete the following tasks:
...
+* Identity and Access Management (IAM) provides a unified approach to managing user identities, services, and access control. IAM authentication requires that an IAM API key is exchanged for a time-limited access token before you can make a request to Cloudant. When the access token expires, the client must obtain a new one from the IAM token service.
+In this exercise, you use the IAM method as the authentication method for the Cloudant service.
+{% endhint %}
+
+* [ ] Click **Create**, as shown in the following figure.
+
+![](../.gitbook/assets/ex3/image007.jpg)
+
+* [ ] After the service is created, you are redirected to the Resource list window, as shown in the following figure. Open the Cloudant service instance overview by clicking the newly created service in the Resource list window. 
+
```
Comment (don-morrison-2000) [inline workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md]: This is a comment on the entire paragraph

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md [hunk 3]
```diff
@@ -87,6 +178,29 @@ Complete the following tasks:
 
 * [ ] From the Cloudant service Overview page, select **Service credentials** from the left pane, as shown in the following figure.
```
Comment (don-morrison-2000) [inline workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md]: This is a comment on a section that did not change

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md (hunk 1 above)
Comment (marcelaadan1) [reply workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md:79]: I can see the final form in VS COde

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md (hunk 2 above)
Comment (marcelaadan1) [reply workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md]: This is a replay to Don's comments on lines 78 - 89

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md [hunk 4]
```diff
@@ -112,39 +397,453 @@ Complete the following tasks:
...
+
+![](../.gitbook/assets/ex3/image089.png)
+
+* [ ] Confirm that you want to delete the service by clicking **Delete**, as shown in the following figure.
+
+![](../.gitbook/assets/ex3/image090.png)
+
+* [ ] Wait until the service is deleted.
 
 ## 🎓Exercise review and wrap-up
```
Comment (marcelaadan1) [inline workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md:763]: This is a comment for the wrap up section lines 761 to 765

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md [hunk 5]
```diff
@@ -65,6 +65,37 @@ Complete the following tasks:
...
 
+* [ ] To complete the creation of the Cloudant service instance, accept the default value for region and keep the **Lite** price plan as the selected plan as shown in the following figure.
+
+![](../.gitbook/assets/ex3/image005.jpg)
+
+* [ ] Accept the default values for service name and resource group.
+* [ ] In **Available authentication methods**, select **Use only IAM**, as shown in the following figure. 
+
+![](../.gitbook/assets/ex3/image006.jpg)
+
```
Comment (marcelaadan1) [inline workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md:77]: Comment on line 76

Comment (marcelaadan1) [reply workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md:77]: Update completed

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md (hunk 2 above)
Comment (marcelaadan1) [reply workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md]: Done

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md (hunk 3 above)
Comment (marcelaadan1) [reply workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md]: Done

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md (hunk 4 above)
Comment (marcelaadan1) [reply workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md:763]: Done

File: workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md [hunk 6]
```diff
@@ -112,39 +399,453 @@ Complete the following tasks:
...
+![](../.gitbook/assets/ex3/image089.png)
+
+* [ ] Confirm that you want to delete the service by clicking **Delete**, as shown in the following figure.
+
+![](../.gitbook/assets/ex3/image090.png)
+
+* [ ] Wait until the service is deleted.
 
 ## 🎓Exercise review and wrap-up
```
Comment (marcelaadan1) [inline workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md:763]: You need to add more info to the wrap up

Comment (marcelaadan1) [reply workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md:763]: OK... Done!

**Review by marcelaadan1 — COMMENTED** [review]
I approve the changes. #1

**Comment by marcelaadan1** [comment]
Merging as admin

**Pull request merged**